server/static/js/index/home_page_processing.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+const TABLE_USERS = '#table_users';
+const TABLE_DEVICES = '#table_device';
+let renderTableDelay = 200;
+
+// Callback function, called on success, for reading device data
```

Comment:
Function declarations should have proper JS docstring (if applicable), so consider changing broad comment to a docstring.